File: lib/layer2_core/dark/e_outline_variant_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class EOutlineVariantDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd46;
      case 'standard': return cstd46;
      case 'blue': return cbd46;
      case 'green': return cgd46;
      case 'red': return crd46;
      case 'nothing': return cnod46;
      default: return cstd46;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/EOutlineVariantDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object EOutlineVariantDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd46
            AppTheme.STANDARD -> R.color.cstd46
            AppTheme.BLUE -> R.color.cbd46
            AppTheme.GREEN -> R.color.cgd46
            AppTheme.RED -> R.color.crd46
            AppTheme.NOTHING -> R.color.cnod46
        }
    }
}
